spec.html

```diff
@@ -25148,6 +25247,10 @@ <h1>Static Semantics: ReferencedBindings</h1>
         <emu-alg>
           1. Return a List containing the first |IdentifierName|.
         </emu-alg>
+        <emu-grammar>ExportSpecifier : ModuleExportName `as` ModuleExportName</emu-grammar>
```

Comment:
This also needs to handle the non‑renaming `ExportSpecifier : ModuleExportName` shorthand.

```suggestion
        <emu-grammar>
          ExportSpecifier :
            ModuleExportName
            ModuleExportName `as` ModuleExportName
        </emu-grammar>
```